csg2csg/MCNPInput.py

```diff
@@ -98,7 +98,7 @@ def __get_importances(self, start_line):
                 return self.__process_importances()
 
             # check for importance keyword
-            if "imp" in self.file_lines[idx]:
+            if self.file_lines[idx].startswith('imp'):
```

Comment:
How about checking the start and end of the line

```suggestion
            if self.file_lines[idx].startswith('imp') or self.file_lines[idx].endswith('imp') :
```

Reply:
having just had a similar issue i found we already had a pr to fix
endswith wont work, 
changing the search to "imp:" makes the most sense

Reply:
```suggestion
            if self.file_lines[idx].startswith('imp:'):
```